src/tdlib/message.rs

```diff
@@ -196,8 +197,11 @@ impl Message {
             .unwrap();
         imp.chat.set(Some(chat));
         imp.forward_info.set(forward_info).unwrap();
-        imp.reply_in_chat_id.set(td_message.reply_in_chat_id);
-        imp.reply_to_message_id.set(td_message.reply_to_message_id);
+        // TODO: Support stories messages
```

Comment:
Rereading this code, I think it's a better idea to just expose the `Message.reply_to` directly (just like in the case of `Chat.block_list` instead of `Chat.is_blocked`), so that we can properly handle replies to stories when we eventually support them.